.github/workflows/rust-checks.yml

```diff
@@ -1,4 +1,5 @@
-name: Cargo fmt
+# Rust specific tests like formatting, clippy, etc
+name: Rust checks
```

Comment:
Does cargo fmt part just check whether the rust files are formatted correctly, or does it actually change them if they are not?
I'm asking because I'm assuming that cargo fmt can actually change files but "Rust checks" makes it sound like it only checks and doesn't actually change anything.
(I don't know if a workflow could actually change what's committed or if that's not possible anyway.)

Reply:
That's correct, the cargo fmt bit will actually push the formatted changes; however the clippy bit won't. How would you call this?

Reply:
I would call it "Cargo fmt and clippy" and remove the "etc.".
If/when we decide to add more, maybe then it's clear what it should be called instead.

Reply:
Right; that was my first thought but I assumed whomever would add a job would forget to update the name. I'm happy with "Cargo fmt and clippy" though. How would you name the file though...?

Reply:
Can it be just `rust.yml`?